tests/unit/test_loaders.py

```diff
@@ -22,12 +22,19 @@
 import contextlib
 import copy
 import os
+import pathlib
+
+try:
+    from zipfile import Path as ZipPath
+except ImportError:
+    from zipp import Path as ZipPath
```

Comment:
What situation will zipfile not be present?

Reply:
`zipfile.Path` only exists as of python 3.8. Since we are only supporting python 3.9+, the exception handling will remain, but instead a dummy `ZipPath` class will be created that raises a `RuntimeError` if its ever initialized